packages/ui-components/src/components/Chart/Table/Table.js

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
The `Chart/Table/Table.js` and `Map/Table/Table.js`are also very similar so the comment above probably applies here as well.

Reply:
Yes this one is more of a concern IMO, it'd be nice if we can find a way to share the code. If you could put an hour into it as a timebox, and if you don't get anywhere, then add a comment to each so that people are at least aware that there might be another place to update if they're e.g. fixing a bug

Reply:
@edmofro - What are your thoughts on me removing the existing Table components from `ui-components` and putting them in psss. And then making this the Table component of ui-components? Alternatively, we could leave in the old table and call this something different such as: DataTable or ReactTable?

Reply:
Hmmmm...well I would say that PSSS tables support a couple of extra things that are quite nice, but this table is probably more what you'd expect from a basic table export. 

Another option is to export them both from ui-components, but this one as `Table` and the PSSS one as something indicative of its enhanced functionality like `ComplexTable`/`NestableTable`/`ExpandingRowTable`.

I'm happy to go with your decision on this, you've been closer to all these different tables than anyone